methods.go

```diff
@@ -935,6 +962,26 @@ func (c *Client) EditTrackerCtx(ctx context.Context, hash string, old, new strin
 	}
 }
 
+// SetQueueingEnabled enable/disable torrent queueing
+func (c *Client) SetQueueingEnabled(enabled bool) error {
+	return c.SetPreferences(map[string]interface{}{"queueing_enabled": enabled})
+}
+
+// SetMaxActiveDownloads set max active downloads
+func (c *Client) SetMaxActiveDownloads(max int) error {
+	return c.SetPreferences(map[string]interface{}{"max_active_downloads": max})
+}
+
+// SetMaxActiveTorrents set max active torrents
+func (c *Client) SetMaxActiveTorrents(max int) error {
+	return c.SetPreferences(map[string]interface{}{"max_active_torrents": max})
+}
+
+// SetMaxActiveUploads set max active uploads
+func (c *Client) SetMaxActiveUploads(max int) error {
+	return c.SetPreferences(map[string]interface{}{"max_active_uploads": max})
+}
```

Comment:
Maybe these should be prefixed with `SetPreferences` like `SetPreferencesQueueingEnabled`.

Should probably have done similar for the torrent priority methods like `SetTorrentMaxPriority` 😄

Reply:
Fixed👍